src/macaron/slsa_analyzer/provenance/loader.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2022 - 2023, Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2022 - 2024, Oracle and/or its affiliates. All rights reserved.
```

Comment:
I wonder why we need to update the copyright year of this file while its content is not changed :thinking:

Reply:
I think this has not been addressed :sweat_smile:

Reply:
I was waiting to do a force push update to fix this issue. Otherwise I think our checks will continue to "fix" the year.

Reply:
Oh I see. Thanks for letting me know. 
Hm that's interesting, I have never encountered this issue before :sweat_smile: